app/scripts/controllers/home.ctrl.js

```diff
@@ -84,7 +87,15 @@ angular.module('yourStlCourts').controller('HomeCtrl', function (Citations,toast
     } else if(optionSelectedMap[ctrl.OptionToSelect.LOCATION]) {
       var names = [];
       ctrl.citationCriteria.municipalityNames.forEach(function(municip){
-        names.push(municip.municipality_name);
+        if (municip.municipality_name == "St. Louis County"){
```

Comment:
None of this new logic has tests. Also - consider putting this logic behind the Municipalities angular service instead